examples/restaurant_demo/restaurant_demo.ipynb

```diff
@@ -0,0 +1,3257 @@
+{
```

Comment:
<p>Setup complete is not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>ok</p>